### ISSUE TITLE
[Bug] Problem with downloading purchased volumes from Viz

### ISSUE THREAD

**Issue by neemanth14** [manga-download/hakuneko#3910]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes/No
Yes
**Describe the bug**
I'm not able to download purchased volumes from viz ( Clipboard Method) It just shows an error in console tab, i'll attach some screenshots below
**To Reproduce**
1. Go to 'Copy the purchased vol link from viz'
2. Click on ' Select the clipboard method"
3. Scroll down to ' You can see that the manga wont load"

**Expected behavior**
I expected to see the manga load as it usually does by using the clipboard method and then for the volume/chapter list to load.
**Screenshots**
https://imgur.com/a/EeMP7nD

**Context (please complete the following information):**
 - OS: [Windows 11]
 - HakuNeko Nightly 8.3.4

**Additional context**

**Comment by neemanth14** [comment]
@playmer hey playmer you asked to ping me here in github, could you check the issue?

**Comment by playmer** [comment]
Just to clarify, is the _only_ issue here the clipboard method? I'm having no issues seeing the volumes/chapters versions of manga in my list and grabbing them.

**Comment by playmer** [comment]
I'm not sure what code path the clipboard goes through but I am seeing that issue as well. I'm just not very familiar with how the clipboard method works to debug it and adjust the code for it.

**Comment by neemanth14** [comment]
> Just to clarify, is the _only_ issue here the clipboard method? I'm having no issues seeing the volumes/chapters versions of manga in my list and grabbing them.

yes , but is there another method to download purchased volumes from viz through HN cause when i try to go to viz library and paste the volume link in HN its not working
I think the one you're talking about is the list we get from the connector but as far as i know we cannot download purchased volumes from the list right?

**Comment by playmer** [comment]
You can, you visit Viz from inside Hakuneko, log in, and then load the Manga list. If you have volumes you've purchased you'll see an entry for that volume list. It can be a little confusing sometimes if the Chapters are also availible for that manga as you might have two entries in the manga list, one for the Volumes and one for the Chapters.

This is the method I use and it's what I tested with.

**Comment by neemanth14** [comment]
> You can, you visit Viz from inside Hakuneko, log in, and then load the Manga list. If you have volumes you've purchased you'll see an entry for that volume list. It can be a little confusing sometimes if the Chapters are also availible for that manga as you might have two entries in the manga list, one for the Volumes and one for the Chapters.
> 
> This is the method I use and it's what I tested with.

ohhh! i'll check that out , thanks

**Comment by neemanth14** [comment]
> You can, you visit Viz from inside Hakuneko, log in, and then load the Manga list. If you have volumes you've purchased you'll see an entry for that volume list. It can be a little confusing sometimes if the Chapters are also availible for that manga as you might have two entries in the manga list, one for the Volumes and one for the Chapters.
> 
> This is the method I use and it's what I tested with.

Hi, i have tried the method you said but its not working. I have Jujutsu Kaisen manga in my library but it doesnt show 2 entries in the list nor does it have the volumes in the entry which has chapters. I'll attach the screenshots, please check them once.
![hakuneko_dnvJ73OR4V](https://user-images.githubusercontent.com/52207756/132009273-d944a33d-ff9f-4a73-afcc-a3efeaed9394.png)
![hakuneko_4vo2ogt7N8](https://user-images.githubusercontent.com/52207756/132009319-61f7dff8-2911-4d9c-b81b-d2c52936ec17.png)

**Comment by playmer** [comment]
Can you show the log when you try to fetch the manga list when logged in?

**Comment by neemanth14** [comment]
> Can you show the log when you try to fetch the manga list when logged i

**Comment by neemanth14** [comment]
> Can you show the log when you try to fetch the manga list when logged in?

Logs? Do you mean the dev tools console error? 
![hakuneko_IIgPTvbsve](https://user-images.githubusercontent.com/52207756/132080912-f1c75f1e-edb7-4ff6-a690-9594335a91d6.png)
![hakuneko_uJJQgqKNlz](https://user-images.githubusercontent.com/52207756/132080918-dc1ef12d-ceab-4fe8-8851-df1ad5cb9567.png)
![hakuneko_sxeiFbWRjG](https://user-images.githubusercontent.com/52207756/132080957-d4f83fbf-8262-4308-89b9-2b2442551165.png)

**Comment by playmer** [comment]
Hmm, I would expect to see some errors when it's trying to parse your library or something...I'm not sure how to fix this without being able to repro. Maybe some additional logging could be put in...but then you'd need to run that locally.

**Comment by neemanth14** [comment]
> Hmm, I would expect to see some errors when it's trying to parse your library or something...I'm not sure how to fix this without being able to repro. Maybe some additional logging could be put in...but then you'd need to run that locally.

If its possible, can you try to connect on AnyDesk for a minute or two?

**Comment by playmer** [comment]
I'm not sure what that would accomplish?

**Comment by neemanth14** [comment]
> I'm not sure what that would accomplish?

We'll be able to reproduce the bug, just for a minute when you're free? Can you also mention the time so that i'll come online

**Comment by playmer** [comment]
I don't just need to see it repro'd. I need to be able to repro it myself so I can debug it and understand what's happening.

**Comment by neemanth14** [comment]
> I don't just need to see it repro'd. I need to be able to repro it myself so I can debug it and understand what's happening.

oh!

**Comment by MikeZeDev** [comment]
Does is works on nightly now that https://github.com/manga-download/hakuneko/pull/5776 is merged?

**Comment by MikeZeDev** [comment]
use haruneko.
